src/containers/GraphPageContainer.ts

```diff
@@ -22,7 +23,7 @@ const mapStateToProps = (state: KialiAppState) => ({
       }
     : null,
   showLegend: state.graph.filterState.showLegend,
-  pollInterval: state.graph.filterState.refreshRate,
+  pollInterval: refreshIntervalSelector(state),
```

Comment:
With respect to the prior "onDuration" drill-down comment, I think that can be removed and we should add the duration state in here, so that changes to duration re-render the graph.